runtime/onert/backend/cpu/ops/TanhLayer.cc

```diff
@@ -34,6 +34,25 @@ TanhLayer::TanhLayer() : _input(nullptr), _output(nullptr)
   // DO NOTHING
 }
 
+void TanhLayer::PopulateLookupTable()
+{
+  const auto input_scale = static_cast<double>(_input->data_scale());
+  const auto input_zero_point = static_cast<int32_t>(_input->data_offset());
+  const auto output_scale = static_cast<double>(_output->data_scale());
+  const auto output_zero_point = static_cast<int32_t>(_output->data_offset());
+  const float inverse_scale = 1 / output_scale;
+  int32_t maxval = std::numeric_limits<uint8_t>::max();
+  int32_t minval = std::numeric_limits<uint8_t>::min();
+  for (int32_t val = minval; val <= maxval; ++val)
+  {
+    const float dequantized = input_scale * (val - input_zero_point);
+    const float transformed = 1.0f / (1.0f + std::exp(-dequantized));
```

Comment:
Please check formula, above expression seems like sigmoid than tanh.

Reply:
Oops..
Thanks for your comment !